README.textile

```diff
@@ -20,9 +20,9 @@ pre. sudo python setup.py install
 
 h5. Usage
 
-Drag a selection in the grid and then click a window to automatically resize it to fill that portion of the screen.
+Show/hide the grid by pressing <code><Ctrl>+<Alt>+D</code> (currently not configurable).
 
-Show/hide the grid by pressing <Alt>R (currently not configurable).
+Drag a selection in the grid and then click a window to automatically resize it to fill that portion of the screen.
 
 Exit the application by right-clicking on its icon in the status bar and selecting Quit.
```

Comment:
I see a little square with 4 inner squares when using ration with xubuntu. Not sure where that's set in the repo.